velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -226,6 +228,20 @@ class DateTimeFunctionsTest : public functions::test::FunctionBaseTest {
         std::vector<VectorPtr>({timestamps, timezones}));
   }
 
+  std::unique_ptr<core::ExecCtx> execCtx_{
+      std::make_unique<core::ExecCtx>(pool_.get(), queryCtx_.get())};
+
+  std::unique_ptr<exec::ExprSet> compile(const core::TypedExprPtr& expr) {
+    return std::make_unique<exec::ExprSet>(
+        std::vector<core::TypedExprPtr>{expr}, execCtx_.get());
+  }
+
+  core::TypedExprPtr currentTimeCall() {
+    std::vector<facebook::velox::core::TypedExprPtr> emptyVectors;
+    return std::make_shared<core::CallTypedExpr>(
```

Comment:
Is it possible to just put {} inline instead of creating a variable for emptyVectors ?

Reply:
@aditi-pandit I tried your suggestion `return std::make_shared<core::CallTypedExpr>(VARCHAR(), {}, "current_time");` and it errored out with the following when I tried to build:
```
[1/2] Building CXX object velox/functions/prestosql/tests/CMakeFiles/velox_functions_test.dir/DateTimeFunctionsTest.cpp.o
FAILED: velox/functions/prestosql/tests/CMakeFiles/velox_functions_test.dir/DateTimeFunctionsTest.cpp.o 
ccache /Library/Developer/CommandLineTools/usr/bin/c++ -DBOOST_ATOMIC_DYN_LINK -DBOOST_ATOMIC_NO_LIB -DBOOST_CONTEXT_DYN_LINK -DBOOST_CONTEXT_NO_LIB -DBOOST_FILESYSTEM_DYN_LINK -DBOOST_FILESYSTEM_NO_LIB -DBOOST_PROGRAM_OPTIONS_DYN_LINK -DBOOST_PROGRAM_OPTIONS_NO_LIB -DBOOST_REGEX_DYN_LINK -DBOOST_REGEX_NO_LIB -DBOOST_SYSTEM_DYN_LINK -DBOOST_SYSTEM_NO_LIB -DBOOST_THREAD_DYN_LINK -DBOOST_THREAD_NO_LIB -DFMT_LOCALE -DFOLLY_HAVE_INT128_T=1 -DGFLAGS_IS_A_DLL=0 -I/Users/minhancao/velox/. -I/Users/minhancao/velox/velox/external/xxhash -I/Users/minhancao/velox/_build/debug/_deps/xsimd-src/include -I/Users/minhancao/velox/_build/debug -isystem /usr/local/include -isystem /Users/minhancao/velox/velox -isystem /Users/minhancao/velox/velox/external -isystem /Users/minhancao/velox/velox/external/duckdb -isystem /Users/minhancao/velox/velox/external/duckdb/tpch/dbgen/include -isystem /usr/local/opt/openssl@1.1/include -isystem /Users/minhancao/velox/_build/debug/_deps/gtest-src/googletest/include -isystem /Users/minhancao/velox/_build/debug/_deps/gtest-src/googletest -isystem /Users/minhancao/velox/_build/debug/_deps/gtest-src/googlemock/include -isystem /Users/minhancao/velox/_build/debug/_deps/gtest-src/googlemock -isystem /Users/minhancao/velox/_build/debug/_deps/gtest-src/include -isystem /Users/minhancao/velox/_build/debug/_deps/gtest-src -mavx2 -mfma -mavx -mf16c -mlzcnt -std=c++17 -mbmi2 -fvisibility=hidden -fvisibility-inlines-hidden -D USE_VELOX_COMMON_BASE -D HAS_UNCAUGHT_EXCEPTIONS -Wall -Wextra -Wno-unused        -Wno-unused-parameter        -Wno-sign-compare        -Wno-ignored-qualifiers        -Wno-range-loop-analysis          -Wno-mismatched-tags          -Wno-nullability-completeness -Werror -g -std=gnu++17 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk -fPIE -MD -MT velox/functions/prestosql/tests/CMakeFiles/velox_functions_test.dir/DateTimeFunctionsTest.cpp.o -MF velox/functions/prestosql/tests/CMakeFiles/velox_functions_test.dir/DateTimeFunctionsTest.cpp.o.d -o velox/functions/prestosql/tests/CMakeFiles/velox_functions_test.dir/DateTimeFunctionsTest.cpp.o -c /Users/minhancao/velox/velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp
/Users/minhancao/velox/velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp:241:12: error: no matching function for call to 'make_shared'
    return std::make_shared<core::CallTypedExpr>(
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/usr/include/c++/v1/__memory/shared_ptr.h:1113:17: note: candidate template ignored: substitution failure [with _Tp = facebook::velox::core::CallTypedExpr]: deduced incomplete pack <std::shared_ptr<const ScalarType<TypeKind::VARCHAR> >, (no value), const char (&)[13]> for template parameter '_Args'
shared_ptr<_Tp> make_shared(_Args&& ...__args)
                ^
1 error generated.
```

Reply:
That's fine. Thanks !

Reply:
make_shared does not support non-explicit initializer lists, which is super annoying :(